data_diff/__init__.py

```diff
@@ -65,6 +65,10 @@ def diff_tables(
     # Maximum size of each threadpool. None = auto. Only relevant when threaded is True.
     # There may be many pools, so number of actual threads can be a lot higher.
     max_threadpool_size: Optional[int] = 1,
+    # Enable printing stats to stdout
+    print_stats: bool = False,
+    # Print stats using json, print_stats should also be True
+    print_json: bool = False,
```

Comment:
Do we really need to print the JSON? We can just return a JSON, and it's very easy to print.